src/Storages/RabbitMQ/StorageRabbitMQ.cpp

```diff
@@ -1117,6 +1113,13 @@ bool StorageRabbitMQ::streamToViews()
     deactivateTask(looping_task, false, true);
     size_t queue_empty = 0;
 
+    if (!checkDependencies(getStorageID()))
```

Comment:
Maybe better to rename streamToView function, I am not sure what would be a good name. It looks like we return true if we want to reschedule & false if otherwise. If you don't have any good suggestions for renaming maybe we can add a comment.

Reply:
agree, may be `tryStreamToViews` and return `true` on successful stream attempt. Seems better.